apps/hvn/src/components/application-from/ApplicationForm.tsx

```diff
@@ -33,28 +34,42 @@ export const ApplicationForm: FC<ApplicationFormProps> = ({ applicationForm }) =
   const [isSubmitting, setIsSubmitting] = useState(false)
 
   const t = useTranslations('Application')
+  const tParticipation = useTranslations('participation')
 
   const [isUnder18State, setIsUnder18State] = useState(true)
+  const [isModalOpen, setIsModalOpen] = useState(false)
 
   const schema = useApplicationFormSchema(isUnder18State)
 
   const form = useForm<ApplicationFormData>({
     defaultValues: {
       acceptConditions: false,
+      acceptEventConditions: false,
       acceptParent: false,
       city: '',
       dateOfBirth: '',
       email: '',
       event: 'hvn_amsterdam',
       fullName: '',
+      otherParticipation: '',
       parentEmail: '',
       parentFullName: '',
       parentPhone: '',
+      participationType: '',
       phone: '',
     },
     resolver: zodResolver(schema),
   })
 
+  const participationOptions = [
```

Comment:
You should add this field in payload form as well.

`apps/hvn/src/utilities/forms/application-form.ts`